src/main/java/org/mule/extensions/jms/internal/source/JmsMessageListener.java

```diff
@@ -91,6 +97,11 @@ public void onMessage(Message message) {
     SourceCallbackContext context = sourceCallback.createContext();
     if (ackMode.equals(TRANSACTED)) {
       sessionManager.bindToTransaction(session);
+      try {
+        context.bindConnection(connection);
```

Comment:
is ok to bind the same connection on multiple onMessage? no sideeffect?

Reply:
previously It was done by the SDK, now by us